src/vip-framework/index.ts

```diff
@@ -130,12 +130,13 @@ export const testVip = (description: string, proposal: Proposal, options: Testin
       await mineUpTo((await ethers.provider.getBlockNumber()) + VOTING_PERIOD + 1);
       const tx = await governorProxy.connect(proposer).queue(proposalId);
       await tx.wait();
+      console.log(await governorProxy.state(proposalId), "state");
```

Comment:
Do we want to keep this? It seems a debug trace. `state` should be always `5` here